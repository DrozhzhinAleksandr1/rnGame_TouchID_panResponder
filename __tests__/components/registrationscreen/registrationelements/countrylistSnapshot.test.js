import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { CountryList } from '../../../../app/components/registrationscreen/registrationelements/countrylist';
import { localization } from '../../../../app/common/localization';

const mock_validObjCountryRegistration = {value: "USA", isValidValue: true};

it('create snapshot CountryList', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<CountryList textTitle={localization.ru} registrationCountry={mock_validObjCountryRegistration}/>);
    expect(toolbarSnapshot).toMatchSnapshot();
});