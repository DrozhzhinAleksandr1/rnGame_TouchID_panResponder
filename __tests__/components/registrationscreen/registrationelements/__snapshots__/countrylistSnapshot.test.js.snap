// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot CountryList 1`] = `
<Component
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
      "width": "100%",
    }
  }
>
  <TouchableWithoutFeedback
    accessibilityLabel="BtnCountryListRegistrationID"
    onPress={[Function]}
    testID="BtnCountryListRegistrationID"
  >
    <Component
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "flexDirection": "row",
          "height": 40,
          "justifyContent": "space-between",
          "marginTop": 15,
          "padding": 0,
          "paddingRight": 15,
          "width": "100%",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={1}
        style={
          Object {
            "color": "#000000",
            "fontFamily": "Roboto-Light",
            "fontSize": 18,
            "lineHeight": 21,
            "paddingLeft": 15,
          }
        }
      >
        USA
      </Text>
      <Polygon />
    </Component>
  </TouchableWithoutFeedback>
  <ShowWarning
    isshowwarning={false}
    text="Не верно указана страна"
  />
</Component>
`;
